pom.xml

```diff
@@ -320,6 +320,11 @@
       <artifactId>grizzled-slf4j_${scala.version.prefix}</artifactId>
       <version>1.0.3</version>
     </dependency>
+
+    <!--
+      Only used directly in tests, but adding test-scope blocks transitive inclusion via Spark, resulting in runtime
```

Comment:
I thought we just un-`test`-scoped this on #577, not because we need it in non-`test`-scope, but because Maven was excluding it as a transitive dep (which Spark and other libraries need) due to its presence in our POM, but then not including it at all in the assembly JAR since it was scoped `test`.

This comment just attempts to explain that state of affairs. lmk if it would be more clear to say something different, or if my understanding of the situation is incorrect.